nvflare/lighter/impl/master_template.yml

```diff
@@ -1334,25 +1324,24 @@ azure_start_cln_sh: |
   then
     echo "Installing and lauching container in $VM_NAME, may take a few minutes."
     scripts=$(cat <<- 'EOF'
-    sudo apt-get update && \
-    sudo DEBIAN_FRONTEND=noninteractive apt-get install -y ca-certificates curl gnupg lsb-release && \
-    sudo mkdir -p /etc/apt/keyrings && \
-    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg && \
-    echo \
-    "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
-    $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null && \
-    sudo apt-get update && \
-    sudo DEBIAN_FRONTEND=noninteractive apt-get install -y docker-ce docker-ce-cli containerd.io
+  sudo apt-get update && \
+  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y ca-certificates curl gnupg lsb-release && \
+  sudo mkdir -p /etc/apt/keyrings && \
+  curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg && \
+  echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null && \
+  sudo apt-get update && \
+  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y docker-ce docker-ce-cli containerd.io
   EOF
-    )
+  )
     az vm run-command invoke \
       --output json \
       --resource-group $RESOURCE_GROUP \
       --command-id RunShellScript \
       --name $VM_NAME \
       --scripts \
       "$scripts" > /tmp/docker_engine.json
-      sudo usermod -aG docker $ADMIN_USERNAME" > /tmp/docker_engine.json
```

Comment:
It seems I only had to remove this line to make the script work.

Reply:
This line 1355 was a bug when we fixed docker installation script into heredoc.  Very likely it's a copy-and-paste error.  Anyway, the single-line format is still what Azure document says.  So we will follow that in this PR.